src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -357,13 +391,45 @@ private void Browser_Loaded(object sender, RoutedEventArgs e)
             LogToDynamoConsole(msg);
         }
 
+        /// <summary>
+        /// Collect the main and modifier key from KeyEventArgs in order to pass
+        /// that data to eventDispatcher (located in library.html) which is responsible
+        /// for binding KeyDown events between dynamo and webview instances
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        ///
```

Comment:
do these comments belong to some other function?

Reply:
These comments belong to `Browser_KeyDown(object sender, KeyEventArgs e)` I've moved them close to the function and leave the enums above